django/test/runner.py

```diff
@@ -47,6 +91,9 @@ def add_arguments(cls, parser):
         parser.add_argument('-r', '--reverse', action='store_true', dest='reverse',
             default=False,
             help='Reverses test cases order.')
+        parser.add_argument('-d', '--debug-sql', action='store_true', dest='debug_sql',
```

Comment:
As far as I can tell we don't actually document these short options (and don't include them in `./runtests.py` either), so I'd be pretty keen on making it consistent one way or another. Whatever we do should be consistent across all `manage.py` commands though.